### PR TITLE
Migrate application to a mix task

### PR DESCRIPTION
**Pull request by dany-kun**
Also upgrade deps

### CODE REVIEW

File: lib/mix/tasks/playlist.ex [hunk 1]
```diff
@@ -1,7 +1,8 @@
-defmodule App do
-  use Application
+defmodule Mix.Tasks.Playlist do
+  use Mix.Task
```
Comment (sabiwara) [inline lib/mix/tasks/playlist.ex]: I don't think you want to remove you application, which defines the supervision tree and belongs there. You should probably just remove the last call.

The mix task is a nice way to run your one-off thing 👍
Another idea could be to run `mix run -e Core.create_playlist` directly ([doc](https://hexdocs.pm/mix/Mix.Tasks.Run.html)).

Comment (dany-kun) [reply lib/mix/tasks/playlist.ex]: I see, updated. Thanks!

**Review by sabiwara — APPROVED** [review]
LGTM! Just a question : you didn’t update your run action yaml, is this on purpose or did you forget ?

**Comment by dany-kun** [comment]
> LGTM! Just a question : you didn’t update your run action yaml, is this on purpose or did you forget ?

Both :). I should have updated it i this PR indeed but delaying/procrastinating as it will come in the next PR with other changes:
https://github.com/dany-kun/roule-avec-spot/pull/15

**Pull request merged**